website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -7639,6 +7639,8 @@ In this section, we will use [KafkaIO.Read](https://beam.apache.org/releases/jav
 
 #### 13.1.1. Creating cross-language Java transforms
 
+For Beam versions 2.60.0+, please follow [this guide](sdks/python-custom-multi-language-pipelines-guide.md) instead.
```

Comment:
Does this apply to the whole section or just `13.1.1.2`? Do we need to recommend away from `JavaExternalTransform` for cases where it works?

Reply:
Also, should we update this section to recommend the new way (even if its just linking to the full doc) by default, and just link to the legacy page for <2.60.0 instead of leaving all the content here?